docs/sql-ref-syntax-aux-show-create-table.md

```diff
@@ -26,7 +26,7 @@ license: |
 ### Syntax
 
 ```sql
-SHOW CREATE TABLE table_identifier
+SHOW CREATE TABLE table_identifier [ AS SERDE ]
```

Comment:
How about adding a new example with `AS SERDE`

Reply:
ok, I will do it.